getting-started/tables-hypertables.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Tables and hypertables
-excerpt: Hypertables are PostgreSQL tables designed to boost the performance of your real-time analytical queries. Create a table and then convert it into a hypertable in Timescale Console
+excerpt: Create tables and hypertables in your Timescale account
```

Comment:
You sure about this?

Reply:
No, that's a merge error because of different branches. Then again, this table is not going to exist anymore, is it.